big_tests/tests/push_integration_SUITE.erl

```diff
@@ -88,12 +97,27 @@ end_per_suite(Config) ->
     mongoose_push_mock:stop(),
     escalus:end_per_suite(Config).
 
+init_per_group(inbox_msg_notifications, Config) ->
+    rpc(mod_muc_light_db_backend, force_clear, []),
+    case (not ct_helper:is_ct_running()) orelse mongoose_helper:is_rdbms_enabled(domain()) of
```

Comment:
What does `not ct_helper:is_ct_running()` check? Common sense would suggest CT is running, when code from a CT suite is executed, but there might be some tricky stuff happening here.